server_admin/topics/users/delete-user.adoc

```diff
@@ -1,14 +1,14 @@
-[deleting-users]
-= Deleting Users
+[id="proc-deleting-users_{context}"]
+= Deleting users
```

Comment:
I am thinking it would be good to standardize on singular, so this would be "Deleting a user" and the anchor would be [id="proc-deleting-user_{context}"]